test/tap/tests/mysql_hostgroup_attributes_config_file-t.cpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file mysql_hostgroup_attributes_config_file-t.cpp
+ * @brief Reading and saving 'mysql_hostgroup_attributes' table from configuration file:
+ *   1. Correct values should be inserted into 'mysql_hostgroup_attributes' table.
+ *   2. Correct saving of mysql_hostgroup_attributes into config file.
+ *   3. Delete all the values from 'mysql_hostgroup_attributes' table
+ *   4. Correct load of 'mysql_hostgroup_attributes' table values from config file
+ */
+
+#include <cstring>
+#include <string>
+#include <fstream>
+#include <unistd.h>
+
+#include "mysql.h"
+#include "mysqld_error.h"
+
+#include "json.hpp"
+
+#include "tap.h"
+#include "utils.h"
+#include "command_line.h"
+
+using nlohmann::json;
+using std::string;
+using std::fstream;
+
+int save_and_read_mysql_hostgroup_attributes_from_config(MYSQL* admin) {
+	json j_servers_defaults { { "weight", 100 }, { "max_connections", 10000 }, { "use_ssl", 1}, };
+
+	// To run this test locally copy a ProxySQL config file into this path.
+	string save_config_query = {"SAVE CONFIG TO FILE /var/lib/jenkins//scripts/"
+								"docker-mysql-proxysql/conf/proxysql/proxysql.cnf"};
+
+	const string INSERT_QUERY {
```

Comment:
The test is quite complex in nature, but then it fails to "test enough" .
In this case it only insert one row , and only 3 columns out of 12 .
I think we should test all the 12 columns , and create more than just one hostgroup with (of course) different settings.

Reply:
Thanks fix it.